CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 Changes by Version
 ==================
 
-2.16.1 (unreleased)
--------------------
-
-- Nothing yet.
-
+2.16.1 (2019-08-30)
+-------------------
+- Run crossdock with all-in-one instead of Cassandra (#420) <Yuri Shkuro>
+- Add a flag for firehose mode (#419) <Prithvi Raj>
+- Update default sampling rate and server url (#414, #413) <Bryan Boreham>
```

Comment:
Split these
  - Default sampling server URL to agent
  - Update default sampling rate when sampling strategy is refreshed